Geometry/HGCalMapping/interface/HGCalMappingTools.h

```diff
@@ -81,15 +83,19 @@ namespace hgcal {
 
         key.first = i;
 
+        key.first = i;
+
```

Comment:
Repetition:
```suggestion
```

Reply:
Fixed in https://github.com/cms-sw/cmssw/pull/45665/commits/ff9b49a245a7da8d39c3c3322754173ffe9f4fac.